templates/example_cell_clustering.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>I don't see a None here anymore, I don't think this comment applies</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
This should be `None`, I just forgot to reassign it back to `None` since I was testing it locally.